sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 Xerox Corporation
+	1996-1999 Silicon Graphics
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
+	2007 NEC LE-IT
+	2016 Marek Vasut"
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libopsVersion="1.1.1"
+libgplVersion="1.1.2"
+
+libopsVersionCompat="$libopsVersion compat >= ${libopsVersion%%.*}"
+libgplVersionCompat="$libgplVersion compat >= ${libgplVersion%%.*}"
```

Comment:
There should be no space between `libVersion` and `libVersionCompat` as in the [Guidelines][0].

Also, as a suggestion, we usually use camelCase for user variables, so I suggest we stick to that with this recipe.

```diff
-libopsVersion="1.1.1"
-libgplVersion="1.1.2"
-
-libopsVersionCompat="$libopsVersion compat >= ${libopsVersion%%.*}"
-libgplVersionCompat="$libgplVersion compat >= ${libgplVersion%%.*}"
+libVersion="1.1.1"
+libGplVersion="1.1.2"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libGplVersionCompat="$libGplVersion compat >= ${libGplVersion%%.*}
```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

Reply:
Shall we add a small exception, to allow an empty line between the `libVersion`s and the `libVersionCompat`s, **_if_** there is more than one shared library?
(I think it looks nicer with that space, but I'm also OK without it.)